apps/emqx/src/emqx_persistent_session_ds.erl

```diff
@@ -440,10 +440,6 @@ del_subscription(TopicFilter, DSSessionId) ->
 %%--------------------------------------------------------------------
 
 create_tables() ->
-    ok = emqx_ds:open_db(?PERSISTENT_MESSAGE_DB, #{
```

Comment:
This database is created in `emqx_persistent_message:init()`, no need to do it twice.